### DISCUSSION TITLE
Error loading webview.

### DISCUSSION THREAD

**Discussion by sergei-hhh** [community/community#17421] [Codespaces]
**Error loading webview.**

I'm trying to open my html page on the web using the HTML Preview plugin, but it gives an error. I did the same in the app and it works great.

![2022-05-28 (3)](https://user-images.githubusercontent.com/56474951/170811883-3abcb0e7-fc0d-4810-a15c-4a921582f875.png)

**Comment by katorly** [comment]
That's a common issue. I resolved this by using extensions like 'Live Server'.

**Comment by joshspicer** [comment]
Hi @sergei-hhh!

I see `the user denied permission to service workers` from the error you shared.  The Codespaces web client uses browser ['service workers'](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers) to render previews.  Opera is blocking the service worker along with its Ad Blocking/tracker-blocking features. 

It seems [some people](https://github.community/t/the-user-denied-permission-to-use-service-worker-when-using-codespaces/195324) have had success by allowing third party cookies for the `*.github.dev` domain. 

Alternatively, you should be able to connect to your Codespace via VS Code or the `gh` CLI, or by using another browser (Edge/Chrome/Safari) on its default settings.
